packages/webcert/src/store/api/apiMiddleware.ts

```diff
@@ -28,8 +29,13 @@ const handleApiCallBegan: Middleware = ({ dispatch }: MiddlewareAPI) => () => as
       method,
       data,
       withCredentials: true,
+      headers: addLaunchIdIfPresent(),
```

Comment:
Maybe rename "addLaunchIdIfPresent()" to something "getHeaders()" and then within the method add launchId if it exists?